contrib/linearize/linearize-data.py

```diff
@@ -17,7 +17,6 @@
 import time
 import glob
 from collections import namedtuple
-from binascii import unhexlify
```

Comment:
missing changes in `contrib/signet/miner` - should be partial https://github.com/bitcoin/bitcoin/pull/22633